jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/CloseStatus.java

```diff
@@ -211,6 +211,11 @@ public boolean isAbnormal()
         return !isOrdinary(code);
     }
 
+    public void initCause(Throwable cause)
```

Comment:
I would much rather keep CloseStatus immutable and replace the entire object than have it partially immutable.
Why isn't the cause known when it is created? feels wrong!